.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+name: Build and upload binaries
+
+on:
+  repository_dispatch:
+    types: [tag-created]
+
+jobs:
+  release:
+    name: Build and release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            artifact_name: manta-signer
+            asset_name: manta-signer-linux-amd64
+          - os: windows-2019
+            artifact_name: manta-signer
+            asset_name: manta-signer-windows
+          - os: macos-latest
+            artifact_name: manta-signer
+            asset_name: manta-signer-macos-amd64
+    #      - os: macos-10.15
+    #        artifact_name: manta-signer
+    #        asset_name: manta-signer-macos-amd64
```

Comment:
Does MacOS 10 build work? Would be nice to include this if possible, or even build for MacOS 10 instead of MacOS latest, since I think it is always forwards compatible

Reply:
yeah, I'll uncomment it.